### PR TITLE
set up the file strucutre

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
Relates #4

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "antd": "^3.16.3",
+    "axios": "^0.18.0",
+    "http-proxy-middleware": "^0.19.1",
+    "react": "^16.8.2",
+    "react-autosuggest": "^9.4.3",
```
Comment (thejoefriel) [inline client/package.json]: Do we need react-autosuggest? Perhaps remove this for now?

File: client/src/index.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+pre {
```
Comment (thejoefriel) [inline client/src/index.css]: Has this file been set up to load after antd so it doesn't get overwritten by antd's own css set up?

File: client/package.json (hunk 1 above)
Comment (RamyAlshurafa) [reply client/package.json]: oops, will remove it

File: client/src/index.css (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/index.css]: @thejoefriel 
im thinking to clean it and keep the `*` selector
what do you think?

**Pull request merged**